octoprint_mrbeamdoc/__version.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "0.1.2"
+__version__ = "1.0.0"
```

Comment:
do we still need this ?
@Josef-MrBeam  I beleive we discussed the version topic and you confirmed that it could be written to the version file and the version to be used is from the tag name.

Reply:
Yes we can use the octoprint way for this, octoprint is using versioneer for this.
https://github.com/OctoPrint/OctoPrint/blob/master/src/octoprint/_version.py you have to change the setup py to use the versioneer
https://github.com/OctoPrint/OctoPrint/blob/master/setup.py
https://github.com/OctoPrint/OctoPrint/blob/master/versioneer.py

Reply:
ok I don't really understand how this works. But if it is such a small change we can have a call about it tomorrow 
@ahmed-mrbeam @Josef-MrBeam